packages/query-core/src/queriesObserver.ts

```diff
@@ -46,11 +47,12 @@ export class QueriesObserver<
   constructor(
     client: QueryClient,
     queries: Array<QueryObserverOptions<any, any, any, any, any>>,
-    _options?: QueriesObserverOptions<TCombinedResult>,
+    options?: QueriesObserverOptions<TCombinedResult>,
```

Comment:
yes because we update it here:

https://github.com/TanStack/query/blob/f6fdbe2cd49c2757e4c3fea5d7521c2304f6ee63/packages/query-core/src/queriesObserver.ts#L92